### DISCUSSION TITLE
how to migrate chatwoot v2.5.0 to latest without any data loss

### DISCUSSION THREAD

**Discussion by mdsami** [chatwoot/chatwoot#8587] [Self hosted]
how to migrate chatwoot v2.5.0 to latest version without any data loss in same linux vm . help me please

**Comment by sojan-official** [comment]
- Backup Data: Before starting, ensure you have a complete backup of your Chatwoot instance, including the database and files.
- Update in Increments: Upgrade your Chatwoot installation one or two versions at a time. This approach helps in managing changes and troubleshooting if issues arise.
- Keep an eye on logs to see if anything is missing.
